src/components/map/use-map-instance.ts

```diff
@@ -35,7 +35,9 @@ export function useMapInstance(
   const {
     id,
     defaultBounds,
+    // @ts-expect-error TS complains that it's not defined in MapProps
```

Comment:
please don't commit those – there's nothing wrong with tests failing in draft-PRs, and it's running risk of missing one of them at some point.